packages/web-components/src/components/cbp-usa-banner/cbp-usa-banner.tsx

```diff
@@ -0,0 +1,86 @@
+import { Component, Prop, Element, Event, EventEmitter, Host, h } from '@stencil/core';
+
+@Component({
+  tag: 'cbp-usa-banner',
+  styleUrl: 'cbp-usa-banner.scss'
+})
+export class CbpUsaBanner {
+
+  @Element() host: HTMLElement;
+
+  /** When set, specifies that the banner is open */
+  @Prop({ reflect: true }) open: boolean;
+
+  // /** Specifies the context of the component as it applies to the visual design and whether it inverts when light/dark mode is toggled. Default behavior is "light-inverts" and does not have to be specified. */
+  // @Prop({ reflect: true }) context: "light-inverts" | "light-always" | "dark-inverts" | "dark-always";
+  
+  /** Supports adding inline styles as an object */
+  @Prop() sx: any = {};
+  
+  /** A custom event emitted when the banner link control is activated. */
+  @Event() bannerLinkClick: EventEmitter;
+  handleClick(e) {
+    e.preventDefault();
+    this.open = !this.open;
+    this.bannerLinkClick.emit({
+      host: this.host,
+      open: this.open,
+    });
+  }
+
+  render() {
+    return (
+      <Host>
+        <img src="../assets/images/us_flag_small.png" height="11" width="16" alt="usa logo"/>Official website of the United State government. 
+        <cbp-link class='cbp-usa-banner-expand' onClick={() => this.handleClick(event)} href='#'>
```

Comment:
I think this is usually a button (ghost button looks like a link). It's missing all accessibility, which you would find in the vanilla version.

Reply:
removed cbp-link and replaced with cbp-button (ghost)